packages/@webex/plugin-meetings/src/meeting/util.ts

```diff
@@ -107,6 +107,10 @@ MeetingUtil.joinMeeting = (meeting, options) => {
       preferTranscoding: !meeting.isMultistream,
       asResourceOccupant: options.asResourceOccupant,
       breakoutsSupported: options.breakoutsSupported,
+      locale: options.locale,
+      deviceCapabilities: options.deviceCapabilities,
```

Comment:
Don't need add the deviceCapabilities parameter

Reply:
Same as above